tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -754,12 +745,22 @@ public:
 
    void Pack(void *dst, void *src, std::size_t count) const final
    {
-      BitPacking::PackBits(dst, reinterpret_cast<const float *>(src), count, fBitsOnStorage);
+#if R__LITTLE_ENDIAN == 0
+      auto bswapped = std::make_unique<std::uint32_t[]>(count);
+      CopyBswap<sizeof(std::uint32_t)>(bswapped.get(), src, count);
```

Comment:
maybe this could be done in place? Not sure if we are allowed to modify `src` here...